frontend/src/components/NavBar/SideBar.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import {
+  Box,
+  CloseButton,
+  Flex,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import {
+  FiHome,
+  FiCompass,
+  FiSettings,
+  FiStar,
+  FiTrendingUp,
+} from 'react-icons/fi';
+import { LinkWithIconProps, SideBarProps } from '@/types/props';
+import SideBarChild from './SideBarChild';
+
+const LinkItems: LinkWithIconProps[] = [
+  { name: 'Home', icon: FiHome },
+  { name: 'Trending', icon: FiTrendingUp },
+  { name: 'Explore', icon: FiCompass },
+  { name: 'Favourites', icon: FiStar },
+  { name: 'Settings', icon: FiSettings },
+];
+
+function SideBar({ onClose, ...rest }: SideBarProps) {
+  return (
+    <Box
+      transition="3s ease"
+      bg={useColorModeValue('white', 'gray.900')}
```

Comment:
Some sub-components of the NavBar uses the same color scheme i.e. bg, borderRightColor. If we are enforcing the same color scheme, would it be better to pull them out into a color scheme css file?